.github/workflows/build-wheel-release-upload.yml

```diff
@@ -0,0 +1,17 @@
+name: Release (GitHub/PyPI)
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
```

Comment:
This will make a release for any tag I guess so we need a better Regen. Is this still testing or I should be reviewing this for production?

Reply:
Still testing, sorry I forgot to close this pull request will close it rn

Reply:
```
on:
  push:
    tags:
      - '[0-9]+.[0-9]+.[0-9]+*'  # Match tags that start with digit.digit.digit and may have additional qualifiers
```

The above may work